locations/spiders/alh_hotels_au.py

```diff
@@ -1,7 +1,21 @@
+from locations.categories import Categories, apply_category
 from locations.storefinders.storelocatorwidgets import StoreLocatorWidgetsSpider
 
 
 class ALHHotelsAUAUSpider(StoreLocatorWidgetsSpider):
     name = "alh_hotels_au"
     item_attributes = {"brand": "ALH Hotels", "brand_wikidata": "Q119159708"}
     key = "41fdc8f98fc2738172250baa676b369d"
+
+    def parse_item(self, item, location):
+        if "hotel" in item["name"].lower() or "inn" in item["name"].lower():
+            apply_category(Categories.HOTEL, item)
+        elif "tavern" in item["name"].lower():
+            apply_category(Categories.RESTAURANT, item)
+        elif "pub" in item["name"].lower() or "grill" in item["name"].lower():
+            apply_category(Categories.PUB, item)
+        elif "bar" in item["name"].lower():
+            apply_category(Categories.BAR, item)
+        elif "cafe" in item["name"].lower():
```

Comment:
Can you do the `item["name"].lower()` once so we save some CPU cycles?

```suggestion
        item_name = item["name"].lower()
        if "hotel" in item_name or "inn" in item_name:
            apply_category(Categories.HOTEL, item)
        elif "tavern" in item_name:
            apply_category(Categories.RESTAURANT, item)
        elif "pub" in item_name or "grill" in item_name:
            apply_category(Categories.PUB, item)
        elif "bar" in item_name:
            apply_category(Categories.BAR, item)
        elif "cafe" in item_name:
```

Reply:
Done.